content/manuals/engine/logging/drivers/awslogs.md

```diff
@@ -127,6 +127,30 @@ $ docker run \
 > Your AWS IAM policy must include the `logs:CreateLogGroup` permission before
 > you attempt to use `awslogs-create-group`.
 
+### awslogs-create-stream
+
+By default, the log driver will attempt to create the AWS CloudWatch Logs stream to be used
+for container log persistence.
+
+To disable AWS CloudWatch Logs stream creation, set `awslogs-create-stream` to `false`.
```

Comment:
Also non-blocking; try to use active voice where possible; e.g.  this could be "Set `awslogs-create-stream` to `false` to disable AWS CloudWatch Logs stream creation."